docs/persona_paths/manage_products.md

```diff
@@ -11,6 +11,7 @@ your stock and customers can easily browse your catalogs.
 
 [[= cards([
     "pim/create_product_types",
+    "pim/create_product",
```

Comment:
That's a good change, but it is outside the scope of this task. Also, the PR with that change is not merged yet, you can add this change there.